exiv2/README.md

```diff
@@ -0,0 +1,21 @@
+# Generate tags.js and index.d.ts
```

Comment:
I'm not a lawyer, but my understanding is that parsing tags from documentation helps us avoid exiv2's `GPL-2.0-or-later` *code* licensing, so this is a sound approach, thank you.